config/presets/smart_access_brands.json.erb

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Inferno Example SMART Access Brands Bundle",
+  "id": "smart_access_brands_bundle_example_inferno",
+  "test_suite_id": "smart_access_brands",
+  "inputs": [
+    {
+      "name": "smart_access_brands_bundle_url",
```

Comment:
```suggestion
      "name": "smart_access_brands_publication_url",
```

I think you forgot to change this after the update to allow users to paste in bundle contents.